en/mozillavpn.xliff

```diff
@@ -419,203 +427,237 @@
+      <context-group purpose="location"><context context-type="linenumber">109</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewUpdate.qml</context><context context-type="linenumber">173</context></context-group>
     </trans-unit>
   </body></file>
   <file datatype="plaintext" original="../src/ui/views/ViewOnboarding.qml" source-language="en" target-language="en-US"><body>
     <trans-unit id="vpn.onboarding.skip">
       <source xml:space="preserve">Skip</source>
       <target xml:space="preserve" />
-      <context-group purpose="location"><context context-type="linenumber">63</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">66</context></context-group>
     </trans-unit>
     <trans-unit id="vpn.onboarding.next">
       <source xml:space="preserve">Next</source>
       <target xml:space="preserve" />
-      <context-group purpose="location"><context context-type="linenumber">72</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">75</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/views/ViewSubscriptionNeeded.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.subscription.title">
+      <source xml:space="preserve">Subscribe for %1/mo</source>
+      <target xml:space="preserve" />
+      <context-group purpose="location"><context context-type="linenumber">71</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.subscription.moneyBackGuarantee">
+      <source xml:space="preserve">30-day money-back guarantee</source>
+      <target xml:space="preserve" />
+      <context-group purpose="location"><context context-type="linenumber">73</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.subscription.featureTitle1">
+      <source xml:space="preserve">No activity logs</source>
+      <target xml:space="preserve" />
+      <context-group purpose="location"><context context-type="linenumber">107</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.subscription.featureSubtitle1">
+      <source xml:space="preserve">We're Mozilla. We're on your side.</source>
+      <target xml:space="preserve" />
+      <context-group purpose="location"><context context-type="linenumber">109</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.subscription.featureTitle2">
+      <source xml:space="preserve">No one can track you</source>
+      <target xml:space="preserve" />
+      <context-group purpose="location"><context context-type="linenumber">115</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.subscription.featureSubtitle2">
+      <source xml:space="preserve">We encrypt your entire device.</source>
```

Comment:
Possible note for copy: this sounds ominous… (you encrypt all connections on the device, not the device itself)